unity-client/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/AssetBundles/AB/AssetPromise_AB.cs

```diff
@@ -164,12 +147,11 @@ protected IEnumerator LoadAssetBundleWithDeps(string baseUrl, string hash, Actio
                 yield return promise;
             }
 
-            AssetBundle assetBundle = DownloadHandlerAssetBundle.GetContent(assetBundleRequest);
+            AssetBundle assetBundle = DownloadHandlerAssetBundle.GetContent(asyncOp.webRequest);
+            asyncOp.Dispose();
 
             if (assetBundle == null || asset == null)
             {
-                assetBundleRequest.Abort();
```

Comment:
Why this was removed and the dispose moved outside the if statement? Are you 100% that this can't introduce regressions?

Reply:
If you notice, right before this conditional we are getting the downloaded content of the request:

`AssetBundle assetBundle = DownloadHandlerAssetBundle.GetContent(asyncOp.webRequest);`

As the request operation will not be used anymore from this point, it is the perfect place where dispose it. The old code not only was not making a dispose of the request, but was aborting a request that had already finished... and that doesn't make sense.

The current code has been tested and for me is more performant in terms of memory felease.

Reply:
Amazing.